releasenotes/notes/drop-legacy-inbound-passthrough.yaml

```
@@ -4,3 +4,4 @@ area: traffic-management
 releaseNotes:
 - |
   **Removed** the `PILOT_ENABLE_INBOUND_PASSTHROUGH` setting, which has been enabled-by-default for the past 8 releases.
+  This feature can now be enabled using a new Inbound Traffic Policy Mode (https://github.com/istio/api/blob/9911a0a6990a18a45ed1b00559156dcc7e836e52/mesh/v1alpha1/config.proto#L203).
```

```suggestion
  This feature can now be disabled using a new Inbound Traffic Policy Mode (https://github.com/istio/api/blob/9911a0a6990a18a45ed1b00559156dcc7e836e52/mesh/v1alpha1/config.proto#L203).
```

Pod-IP binding is still enabled-by-default, no? So only people seeking to disable the default of Pod-IP binding would need to do this.

---

yes. I will change